### DISCUSSION TITLE
Questions for release meet-up June 2022

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#259] [Answers & Questions]
Hello!

Cannot wait for you to go through all the new features :)

When you go through, would it be possible to cover these things:
- Any new ops/techniques related to MeshInstancerFromTexture workflow (like f.ex. a usecase for the TexMathModulo, InstancedModulo, RgbCurlNoise, ...)
- Is it possible to texture extruded geo with different materials on different sides (in one operation)?
- VertTexTransform

**Comment by steam0r** [comment]
you can take a look at what is gonna be in there already on https://dev.cables.gl/changelog (work in progress) but we will definitely cover all/most of the new ops and features in that session :)

**Reply by steam0r** [reply]
https://cables.gl/changelog by now ;)

**Reply by franckaubry** [reply]
And have of course read through it all by now 😜
